<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UICCurveConfigurationDialog</class>
 <widget class="QWidget" name="UICCurveConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>655</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FaceTrackNoIR tracking curves</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/facetracknoir.png</normaloff>images/facetracknoir.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ccc;</string>
  </property>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>625</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>886</x>
     <y>625</y>
     <width>69</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ccc;</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>6</number>
   </property>
   <widget class="QWidget" name="tabWidgetPage1">
    <attribute name="title">
     <string>Yaw</string>
    </attribute>
    <widget class="QFunctionConfigurator" name="rxconfig" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>921</width>
       <height>240</height>
      </rect>
     </property>
     <property name="maxInputEGU" stdset="0">
      <number>180</number>
     </property>
     <property name="maxOutputEGU" stdset="0">
      <number>180</number>
     </property>
     <property name="pixPerEGU_Input" stdset="0">
      <number>5</number>
     </property>
     <property name="colorBezier" stdset="0">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
     <property name="colorBackground" stdset="0">
      <color>
       <red>240</red>
       <green>240</green>
       <blue>240</blue>
      </color>
     </property>
    </widget>
    <widget class="QCheckBox" name="rx_altp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>166</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Use alternative</string>
     </property>
    </widget>
    <widget class="QFunctionConfigurator" name="rxconfig_alt" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>975</width>
       <height>206</height>
      </rect>
     </property>
     <property name="maxInputEGU" stdset="0">
      <number>180</number>
     </property>
     <property name="maxOutputEGU" stdset="0">
      <number>180</number>
     </property>
     <property name="pixPerEGU_Input" stdset="0">
      <number>5</number>
     </property>
     <property name="colorBezier" stdset="0">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
     <property name="colorBackground" stdset="0">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage2">
    <attribute name="title">
     <string>Pitch</string>
    </attribute>
    <widget class="QFunctionConfigurator" name="ryconfig" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>595</width>
       <height>240</height>
      </rect>
     </property>
     <property name="maxInputEGU" stdset="0">
      <number>90</number>
     </property>
     <property name="maxOutputEGU" stdset="0">
      <number>90</number>
     </property>
     <property name="pixPerEGU_Input" stdset="0">
      <number>10</number>
     </property>
     <property name="pixPerEGU_Output" stdset="0">
      <number>2</number>
     </property>
     <property name="colorBezier" stdset="0">
      <color>
       <red>0</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </property>
     <property name="colorBackground" stdset="0">
      <color>
       <red>240</red>
       <green>240</green>
       <blue>240</blue>
      </color>
     </property>
    </widget>
    <widget class="QCheckBox" name="ry_altp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>199</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Use alternative</string>
     </property>
    </widget>
    <widget class="QFunctionConfigurator" name="ryconfig_alt" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>1013</width>
       <height>240</height>
      </rect>
     </property>
     <property name="maxInputEGU" stdset="0">
      <number>90</number>
     </property>
     <property name="maxOutputEGU" stdset="0">
      <number>90</number>
     </property>
     <property name="pixPerEGU_Input" stdset="0">
      <number>10</number>
     </property>
     <property name="pixPerEGU_Output" stdset="0">
      <number>2</number>
     </property>
     <property name="colorBezier" stdset="0">
      <color>
       <red>0</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </property>
     <property name="colorBackground" stdset="0">
      <color>
       <red>240</red>
       <green>240</green>
       <blue>240</blue>
      </color>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage3">
    <attribute name="title">
     <string>Roll</string>
    </attribute>
    <widget class="QFunctionConfigurator" name="rzconfig" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>595</width>
       <height>240</height>
      </rect>
     </property>
     <property name="maxInputEGU" stdset="0">
      <number>180</number>
     </property>
     <property name="maxOutputEGU" stdset="0">
      <number>180</number>
     </property>
     <property name="pixPerEGU_Input" stdset="0">
      <number>5</number>
     </property>
     <property name="pixPerEGU_Output" stdset="0">
      <number>1</number>
     </property>
     <property name="colorBezier" stdset="0">
      <color>
       <red>0</red>
       <green>0</green>
       <blue>255</blue>
      </color>
     </property>
     <property name="colorBackground" stdset="0">
      <color>
       <red>240</red>
       <green>240</green>
       <blue>240</blue>
      </color>
     </property>
    </widget>
    <widget class="QCheckBox" name="rz_altp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Use alternative</string>
     </property>
    </widget>
    <widget class="QFunctionConfigurator" name="rzconfig_alt" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>595</width>
       <height>240</height>
      </rect>
     </property>
     <property name="maxInputEGU" stdset="0">
      <number>180</number>
     </property>
     <property name="maxOutputEGU" stdset="0">
      <number>180</number>
     </property>
     <property name="pixPerEGU_Input" stdset="0">
      <number>5</number>
     </property>
     <property name="pixPerEGU_Output" stdset="0">
      <number>1</number>
     </property>
     <property name="colorBezier" stdset="0">
      <color>
       <red>0</red>
       <green>0</green>
       <blue>255</blue>
      </color>
     </property>
     <property name="colorBackground" stdset="0">
      <color>
       <red>240</red>
       <green>240</green>
       <blue>240</blue>
      </color>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage4">
    <attribute name="title">
     <string>X</string>
    </attribute>
    <widget class="QFunctionConfigurator" name="txconfig" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>555</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maxInputEGU" stdset="0">
      <number>100</number>
     </property>
     <property name="maxOutputEGU" stdset="0">
      <number>100</number>
     </property>
     <property name="pixPerEGU_Input" stdset="0">
      <number>7</number>
     </property>
     <property name="pixPerEGU_Output" stdset="0">
      <number>2</number>
     </property>
     <property name="colorBezier" stdset="0">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>255</blue>
      </color>
     </property>
     <property name="colorBackground" stdset="0">
      <color>
       <red>240</red>
       <green>240</green>
       <blue>240</blue>
      </color>
     </property>
    </widget>
    <widget class="QCheckBox" name="tx_altp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>228</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Use alternative</string>
     </property>
    </widget>
    <widget class="QFunctionConfigurator" name="txconfig_alt" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>555</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maxInputEGU" stdset="0">
      <number>100</number>
     </property>
     <property name="maxOutputEGU" stdset="0">
      <number>100</number>
     </property>
     <property name="pixPerEGU_Input" stdset="0">
      <number>7</number>
     </property>
     <property name="pixPerEGU_Output" stdset="0">
      <number>2</number>
     </property>
     <property name="colorBezier" stdset="0">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>255</blue>
      </color>
     </property>
     <property name="colorBackground" stdset="0">
      <color>
       <red>240</red>
       <green>240</green>
       <blue>240</blue>
      </color>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage5">
    <attribute name="title">
     <string>Y</string>
    </attribute>
    <widget class="QFunctionConfigurator" name="tyconfig" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>555</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maxInputEGU" stdset="0">
      <number>100</number>
     </property>
     <property name="maxOutputEGU" stdset="0">
      <number>100</number>
     </property>
     <property name="pixPerEGU_Input" stdset="0">
      <number>7</number>
     </property>
     <property name="pixPerEGU_Output" stdset="0">
      <number>2</number>
     </property>
     <property name="colorBezier" stdset="0">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </property>
     <property name="colorBackground" stdset="0">
      <color>
       <red>240</red>
       <green>240</green>
       <blue>240</blue>
      </color>
     </property>
    </widget>
    <widget class="QCheckBox" name="ty_altp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>229</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Use alternative</string>
     </property>
    </widget>
    <widget class="QFunctionConfigurator" name="tyconfig_alt" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>555</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maxInputEGU" stdset="0">
      <number>100</number>
     </property>
     <property name="maxOutputEGU" stdset="0">
      <number>100</number>
     </property>
     <property name="pixPerEGU_Input" stdset="0">
      <number>7</number>
     </property>
     <property name="pixPerEGU_Output" stdset="0">
      <number>2</number>
     </property>
     <property name="colorBezier" stdset="0">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>0</blue>
      </color>
     </property>
     <property name="colorBackground" stdset="0">
      <color>
       <red>240</red>
       <green>240</green>
       <blue>240</blue>
      </color>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage6">
    <attribute name="title">
     <string>Z</string>
    </attribute>
    <widget class="QFunctionConfigurator" name="tzconfig" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>555</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maxInputEGU" stdset="0">
      <number>100</number>
     </property>
     <property name="maxOutputEGU" stdset="0">
      <number>100</number>
     </property>
     <property name="pixPerEGU_Input" stdset="0">
      <number>7</number>
     </property>
     <property name="pixPerEGU_Output" stdset="0">
      <number>2</number>
     </property>
     <property name="colorBezier" stdset="0">
      <color>
       <red>0</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
     <property name="colorBackground" stdset="0">
      <color>
       <red>240</red>
       <green>240</green>
       <blue>240</blue>
      </color>
     </property>
    </widget>
    <widget class="QCheckBox" name="tz_altp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>263</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Use alternative</string>
     </property>
    </widget>
    <widget class="QFunctionConfigurator" name="tzconfig_alt" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>555</width>
       <height>160</height>
      </rect>
     </property>
     <property name="maxInputEGU" stdset="0">
      <number>100</number>
     </property>
     <property name="maxOutputEGU" stdset="0">
      <number>100</number>
     </property>
     <property name="pixPerEGU_Input" stdset="0">
      <number>7</number>
     </property>
     <property name="pixPerEGU_Output" stdset="0">
      <number>2</number>
     </property>
     <property name="colorBezier" stdset="0">
      <color>
       <red>0</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
     <property name="colorBackground" stdset="0">
      <color>
       <red>240</red>
       <green>240</green>
       <blue>240</blue>
      </color>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage7">
    <attribute name="title">
     <string>Options</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>44</y>
       <width>35</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RY</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>34</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>RX</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>75</y>
       <width>33</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>RZ</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pos_rx">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="suffix">
      <string> deg.</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pos_ry">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="suffix">
      <string> deg.</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pos_rz">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>71</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="suffix">
      <string> deg.</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>209</x>
       <y>10</y>
       <width>32</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TX</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pos_tx">
     <property name="geometry">
      <rect>
       <x>249</x>
       <y>10</y>
       <width>152</width>
       <height>22</height>
      </rect>
     </property>
     <property name="suffix">
      <string> cm</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pos_ty">
     <property name="geometry">
      <rect>
       <x>249</x>
       <y>40</y>
       <width>152</width>
       <height>22</height>
      </rect>
     </property>
     <property name="suffix">
      <string> cm</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="pos_tz">
     <property name="geometry">
      <rect>
       <x>248</x>
       <y>71</y>
       <width>154</width>
       <height>22</height>
      </rect>
     </property>
     <property name="suffix">
      <string> cm</string>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>209</x>
       <y>44</y>
       <width>32</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TY</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>208</x>
       <y>75</y>
       <width>35</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TZ</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>192</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Translation compensation</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFunctionConfigurator</class>
   <extends>QWidget</extends>
   <header>qfunctionconfigurator.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>startEngineClicked()</slot>
  <slot>stopEngineClicked()</slot>
  <slot>cameraSettingsClicked()</slot>
 </slots>
</ui>
